tests/test_ag3.py

```diff
@@ -1266,7 +1254,6 @@ def test_gene_cnv_frequencies(contig, cohorts):
 
         assert isinstance(df, pd.DataFrame)
         assert len(df) == len(df_genes)
-        assert df.index.name == "ID"
```

Comment:
Add test on index names.

Reply:
Hi @alimanfoo, I removed the tests as multi-column-index `df.index.name` returns `None`. I will have a play to see what I can find.

Reply:
Ha ha, it's just `.names` !